ConsoleApp1/Program.cs

```diff
@@ -4,6 +4,149 @@ internal class Program
 	{
 		static void Main(string[] args)
 		{
+			//Cross
+			Cross(10);
+
+			//Example for  max value among all the parameters
+			
+			Maximum(3,2);
+
+			int[] mass = {1,5,3};
+			Maximum(mass);
+
+			// Min value …
+
+			int[] massmin = {-1,6,8};
+			Minimum(massmin);
+			Console.ReadLine();
+
+			//out
+			bool sumResult;
+			TrySumIfOdd(4, 5, out sumResult);
+			Console.WriteLine(sumResult);
+			Console.ReadLine();
+
+			//recursion
+			Recrec("ma",2);
+			Console.ReadLine();
+
 		}
-	}
-}
\ No newline at end of file
+
+		//Method that will return max value among all the parameters
+			static int Maximum(int a, int b)
+			{
+				int c;
+				c = a > b ? a:b;
+				Console.WriteLine($"Maximum is {c}");
+				return c;
+			}
+
+			static int Maximum(params int[] arr)
+			{
+				int c = arr[0];
+				foreach (var i in arr)
+				{
+					c = c < i ? i : c;
+				}
+				Console.WriteLine($"Maximum is {c}");
+				return c;
+			}
+
+
+		// min value …
+
+			static int Minimum(int a, int b)
+			{
+				int c;
+				c = a < b ? a:b;
+				Console.WriteLine($"Minimum is {c}");
+				return c;
+			}
+
+			static int Minimum(params int[] arr)
+			{
+				int c = arr[0];
+				foreach (var i in arr)
+				{
+					c = c < i ? c : i;
+				}
+				Console.WriteLine($"Minimum is {c}");
+				return c;
+			}
+
+		//Method TrySumIfOdd that accepts 2 parameters
+		//and returns true if sum of numbers between inputs is odd,
+		//otherwise false, sum return as out parameter
+
+		static void TrySumIfOdd(int a, int b, out bool sum)
+		{
+			sum  = true;
+			sum = (a+b)%2 == 0 ? false: true; 
```

Comment:
you should declare sum as `out int sum` and change return type from `void ` to `bool`
Where bool shows that sum is odd